packages/nullable-array/nullable-array.0.1/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "1.2"
+maintainer: "Pierre Chambart <pierre.chambart@ocamlpro.com>"
+authors: "Pierre Chambart <pierre.chambart@ocamlpro.com>"
+homepage: "https://github.com/chambart/nullable-array"
+bug-reports: "https://github.com/chambart/nullable-array/issues"
```

Comment:
@chambart These URLs still go 404.
The `ocamll-` prefix is missing https://github.com/chambart/ocaml-nullable-array